### PR TITLE
Angularify RgHeader color classes

### PR DESCRIPTION
**Pull request by chrisvxd**
Used a one-time binding approach via the `link` method on the directive to reduce unnecessary watchers.

A thought: this could be abstracted to generically allow for modifiers on all components via a `modifier` attr.

TP https://rockabox.tpondemand.com/entity/11370

### CODE REVIEW

**Comment by ohskylab** [comment]
@chrisvxd Is this DNM only because #394 needed merging first, or was there another reason?

File: src/rg-header/rg-header-directive.js [hunk 1]
```diff
@@ -42,7 +42,13 @@ define([
             restrict: 'E',
             replace: true,
             transclude: true,
-            template: template
+            template: template,
+            link: function (scope, ele, attrs) {
+                scope.modifier = '';
+                if (angular.isDefined(attrs['color'])) {
```
Comment (nwhite89) [inline src/rg-header/rg-header-directive.js]: do we want a default colour? @ohskylab ?

Comment (nwhite89) [inline src/rg-header/rg-header-directive.js]: if not angular point of view GTM

Comment (ohskylab) [inline src/rg-header/rg-header-directive.js]: Thought about it, but reckon it should be explicitly stated due to the way the guidelines work.

There's a neutral, no-modifier fallback colour in styles to cover no-modifier situations. It doesn't look bad but it also doesn't look correct.

Comment (nwhite89) [inline src/rg-header/rg-header-directive.js]: cool that's fine thought probably the case but thought better check :+1: :)

**Pull request merged**